docs/develop.md

```diff
@@ -0,0 +1,65 @@
+# development
+
+If you like Tigre purpose and would like to contribute this page can help you.
+
+Tigre support only python3.7+ and you need version 3.7+ to develop. We recommend that you use `Pyenv` to manage many python enviroments. Basic instructions can be found below.
```

Comment:
Some changings to review:

`Tigre support` -> `Tigre supports`

`python3.7+` -> `Python 3.7+`

`python environments.` -> `Python environments.`